### ISSUE TITLE
Damage Indicators

### ISSUE THREAD

**Issue by SNSnoop** [teeworlds/teeworlds#2141]
Server sending unused ClientID, Angle and Self variables
https://github.com/teeworlds/teeworlds/blob/d18b5c3fec3b4c028b2c10af850206fdec6a1621/src/game/server/gamecontext.cpp#L96-L110

Happened after this(?) https://github.com/teeworlds/teeworlds/commit/029b0accbd7eebc0e3f8d205f62ff6ccf9da77b6

**Comment by Dune-jr** [comment]
That one is on me: PR https://github.com/teeworlds/teeworlds/pull/1716.
So I was supposed to propose some graphical improvements to the damage indicators that convey a bit of information such as the angle from where the hit comes from and possibly with small hull effects for shield damage.

I haven't found a satisfactory, elegant solution before 0.7.3 though, partly because I'm not good at designing teeish graphical stuff (esp. animations) and was a bit on my own with the design bit.

I still have hopes to find a way that fits the minimalistic spirit of Teeworlds, or that someone helps me with that :)

**Comment by oy** [comment]
Yeah, we gonna expand that feature someday :)
Dunno if this should stay open as a reminder? There's also another (self)damage indicator issue.

**Comment by Dune-jr** [comment]
Yeah, we can leave that open, https://github.com/teeworlds/teeworlds/issues/1411 isn't quite the same scope.

**Comment by fokkonaut** [comment]
would be cool if the server can send the angle again, and the client would only automatically (like right now) set the angle if the server doesnt send it

**Comment by Learath2** [comment]
@Dune-jr maybe we could just use the angle again? We could use it to indicate the direction the hit came from which seems to be your intention from #1716

**Comment by Dune-jr** [comment]
@Learath2 Yeah, absolutely!
